packages/api/src/middlewares/auth/index.ts

```diff
@@ -79,4 +81,30 @@ function authPermissionMiddleware(permission?: PermissionRoles[]) {
   };
 }
 
-export { authMiddleware, authPermissionMiddleware };
+function authPointsMiddleware(
+  req: IAuthRequest,
+  res: Response,
+  next: NextFunction,
+) {
+  try {
+    const authorization = req?.headers?.authorization;
```

Comment:
For more secure requests, I ask you to encrypt the key that comes in the authorization header:

use the lib `crypto-js`, we need an encrypt key, please add a new environment var


`
// simple impl.

const auth = CryptoJS.AES.decrypt(
        authorization,
        AUTH_POINTS_HASH,
      ).toString(CryptoJS.enc.Utf8);


const validAuth = auth === AUTH_POINTS
`